android/src/main/java/com/oney/WebRTCModule/WebRTCModule.java

```diff
@@ -964,4 +964,12 @@ public void addListener(String eventName) {
     public void removeListeners(Integer count) {
         // Keep: Required for RN built in Event Emitter Calls.
     }
+
+    @ReactMethod
+    public void setVideoEffect(String id, String name) {
```

Comment:
Please rename this to `mediaStreamTrackSetVideoEffect`, analogously to `mediaStreamTrackSwitchCamera`, and put it next  to it in the file.